utils/addresses.ts

```diff
@@ -0,0 +1,5 @@
+import { Workshop } from "../shared/schemas";
+
+export function workshopToReadableAddress(workshop: Workshop) {
```

Comment:
Can we have some some JSDoc/documentation for this and a return type, and then I think we're good to go! Nice one!

Reply:
All done now :+1: